tests/test_health.py

```
@@ -0,0 +1,136 @@
+            result = await tool_fn(action="diagnose")
+        assert "cpu_sub" in result
+
+    async def test_diagnose_import_error_internal(self) -> None:
+        """_diagnose_subscriptions catches ImportError and returns error dict."""
+        import builtins
+
+        from unraid_mcp.tools.health import _diagnose_subscriptions
+
+        real_import = builtins.__import__
+
+        def fail_subscriptions(name, *args, **kwargs):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `fail_subscriptions`

(ANN202)